src/errors.rs

```diff
@@ -39,7 +38,7 @@ pub enum DeployError {
 
     /// An error occured encoding deployment parameters with the contract ABI.
     #[error("error ABI ecoding deployment parameters: {0}")]
-    Abi(AbiErrorKind),
+    Abi(#[from] AbiError),
```

Comment:
Never seen anything like this before.

Reply:
It's from this great crate https://github.com/dtolnay/thiserror .